router-bridge/src/worker.rs

```diff
@@ -278,6 +281,26 @@ async fn receive(state: Rc<RefCell<OpState>>) -> Result<JsonPayload, anyhow::Err
         .map_err(|e| anyhow::anyhow!("op_receive: couldn't send response {e}"))
 }
 
+#[op(fast)]
+fn op_crypto_get_random_values(state: &mut OpState, out: &mut [u8]) -> Result<(), anyhow::Error> {
+    if out.len() > 65536 {
+        return Err(
+      deno_web::DomExceptionQuotaExceededError::new(&format!("The ArrayBufferView's byte length ({}) exceeds the number of bytes of entropy available via this API (65536)", out.len()))
+        .into(),
+    );
+    }
+
+    let maybe_seeded_rng = state.try_borrow_mut::<StdRng>();
+    if let Some(seeded_rng) = maybe_seeded_rng {
+        seeded_rng.fill(out);
+    } else {
+        let mut rng = thread_rng();
+        rng.fill(out);
+    }
+
+    Ok(())
+}
+
```

Comment:
Can we leverage any unit tests from the other implementation as well?  I know that "this will be tested somewhere indirectly", but it's not merely intuitive to believe it **is** tested here without the test next to it.

Reply:
we have a test in https://github.com/apollographql/federation-rs/blob/main/router-bridge/js-src/test_get_random_values.ts and I checked locally that it worked as expected (function properly exported, etc)

Reply:
Ok, just add a comment to the TS plz.

Reply:
done